src/curve.js

```diff
@@ -297,6 +297,18 @@ export function getCurve(curve) {
     return null;
   }
   if (typeof curve == 'string') {
+    // If the curve is a custom cubic-bezier curve
+    if (curve.indexOf('cubic-bezier') != -1) {
+      const floatExp = '(\\d+\\.?\\d*|\\.\\d+)';
```

Comment:
Looking at this, I think matching the overall expression and parsing inside parens using `split` and `parseFloat` might be easier. E.g. something like this:

```
const match = curve.match(/cubic-bezier\((.+)\)/);
if (match) {
  const values = match[1].split(',').map(parseFloat);
  if (values.length == 4) {
   ...
  }
}
```

This is just an advise. I don't feel too strongly about this part.